build_tools/github/vendor.py

```diff
@@ -0,0 +1,148 @@
+"""Embed vcomp140.dll, VCRUNTIME140.dll and VCRUNTIME140_1.dll.
+
+Note that VCRUNTIME140_1.dll is only required (and available)
+for 64-bit architectures.
+"""
+
+
+import os
+import os.path as op
+import shutil
+import sys
+import textwrap
+
+
+TARGET_FOLDER = op.join("sklearn", ".libs")
+DISTRIBUTOR_INIT = op.join("sklearn", "_distributor_init.py")
+VCOMP140_SRC_PATH = "C:\\Windows\\System32\\vcomp140.dll"
+VCRUNTIME140_SRC_PATH = "C:\\Windows\\System32\\VCRUNTIME140.dll"
+VCRUNTIME140_1_SRC_PATH = "C:\\Windows\\System32\\VCRUNTIME140_1.dll"
```

Comment:
nitpick:

I think those windows filename are case insensitive and we could standardize to lowercase everywhere (both in the filenames and in the comments). WDYT?